cip-0059 draft (#52)
global-synchronizer-foundation/cips: cip-0059/cip-0059.md

## CIP 0059

<pre>
  CIP: CIP 0059
  Title:  Add Woodside AI as an SV of Weight 10
  Author: Eric W Saraniecki 
  Status: Draft 
  Type: Governance 
  Created: 2025-04-21
  Approved: TBD
  License: CC0-1.0
</pre>

## Abstract:

> Add Monstera as an SV of Weight 5

Reviewer: ```suggestion
Add Woodside AI as an SV with weight 10
```

## CIP 0059

<pre>
  CIP: CIP 0059
  Title:  Add Woodside AI as an SV of Weight 10
  Author: Eric W Saraniecki 
  Status: Approved 
  Type: Governance 
  Created: 2025-04-21
  Approved: 2025-05-09
  License: CC0-1.0
</pre>

## Abstract:
Add Woodside AI as an SV with weight 10

## About Applicant:

> * Since early 2025, Woodside AI have been developing an investment thesis for Canton network around Private Credit infrastructure, Fund Admin infrastructure together with Blackstone in addition to bringing some assets to Canton from the Middle East around Mortgages, Payment Securitization and more.

Reviewer: ```suggestion
* Since early 2025, Woodside AI have been developing an investment thesis for Canton network around Private Credit infrastructure and Fund Admin infrastructure, together with Blackstone, in addition to bringing some assets to Canton from the Middle East around Mortgages, Payment Securitization and more.
```

> * Since early 2025, Woodside AI have been working on developing an investment thesis for Canton network around Private Credit infrastructure, Fund Admin infrastructure together with Blackstone in addition to bringing some assets to Canton from the Middle East around Mortgages, Payment Securitization and more.

Reviewer: ```suggestion
* Since early 2025, Woodside AI have been working on developing an investment thesis for Canton network around Private Credit infrastructure, Fund Admin infrastructure together with Blackstone in addition to bringing some assets to Canton from the Middle East around Mortgages, Payment Securitization and more.
```

> * Over the past 4 months, Woodside AI have been working on developing an investment thesis for Canton network around Private Credit infrastructure, Fund Admin infrastructure together with Blackstone in addition to bringing some assets to Canton from the Middle East around Mortgages, Payment Securitization and more.

Reviewer: ```suggestion
* Since early 2025, Woodside AI have been working on developing an investment thesis for Canton network around Private Credit infrastructure, Fund Admin infrastructure together with Blackstone in addition to bringing some assets to Canton from the Middle East around Mortgages, Payment Securitization and more.
```

    * Applicant is required to present a calculation for number of Canton Coin it should earn for meeting the requirements of the milestone
* If the Tokenomics Working Group agrees the milestone has been met and agrees with the calculation, an announcement will be sent via the Tokenomics-Announce mailing List
    * The GSF will update the `extraBeneficiary` to an active PartyID controlled by that Super Validator. 
    * ⅔ of Super Validator Operators will then assign a portion of the Unclaimed Rewards to be minted by the Applicant’s Validator, based on the calculation approved by the Tokenomics working group.
   
* If any milestones and associated rewards are not achieved by the deadline
    * Applicant will be notified they have not met a deliverable by the GSF 
    * Remaining SV Weight assigned to the `extraBeneficiary` SV will be removed from the GSF node configuration, and the total SV weight of the GSF SV node will be reduced by the same amount by a vote of the Super Validators.
    * The Tokenomics Working Group will make a recommendation to the SVs on what to do with the Unclaimed Rewards 
* Applicant is subject to CIP-0045 : SV Operating Requirements

>     * If, at any time, the Applicant has been rewarded SV Weight > 2.5, they are required to operate their SV within 6 months of crossing that Weight

Reviewer: ```suggestion
    * If, at any time, the Applicant has been rewarded SV Weight > 2.5, they are required to operate their SV within 6 months of crossing that Weight. This node will start with an SV weight of zero (0) and may add weights as the SV completes the milestones listed above.  
```
